### ISSUE TITLE
Cannot use usys system to disable factions aspect per world

### ISSUE THREAD

**Issue by killyouslow** [MassiveCraft/MassiveCore#217]
Mcore doesn't do its intended job at all, i've tried a million times to completely disable Factions aspect in separate multiverses and it does not work, i can still use factions commands in multiverses that are not using the aspect Factions, i don't know if that's intentional but if it is then i think its ridiculous, whats the point in the usys system if you can't disable aspects

**Comment by magnusulf** [comment]
You  should use the blacklist/whitelist system in the factions configuration.

**Comment by killyouslow** [comment]
Soo then what is mcore purpose, factions would be effective serverwide without mcore, and if you can whitelist / blacklist certain worlds "which i already have" than someone please tell me what is the point of mcore breaking worlds into different universes and multiverses if you can not disable massive plugin aspects per universe / multiverse !!!!

**Comment by magnusulf** [comment]
The universe system is subject to removal. So it will be removed in a future release.

**Comment by killyouslow** [comment]
Soo your telling me that factions will no longer depend on mcore? they NEED to add a config setting that completely disables factions perworld, i mean no messages, no claiming / powerloss / etc and NO USE OF THE COMMANDS! thats what the old mcore system did thru the use of aspects. why is there no longer a setting for this. mcore and factions = 1 headache after a nother via EVERY UPDATE !!!

- Rant over

**Comment by magnusulf** [comment]
It will still depend on MassiveCore because MassiveCore has a lot of utils required for factions to work. You can disable claiming and powerless in the configuration for factions. You will however still be able to use the commands so when being in a non faction world you can still invite people to your faction and etc (which in my opinion is cool that changing world all the time isn't required, but that is subjective).
So multi world support is there, if you want more options please specify them and we will see what we can do :)

**Comment by oloflarsson** [comment]
@killyouslow - You just complain all the time. Uninstall the plugin if you don't like it and move on. Or start talking to developers in a mature and respectul manner, please.

**Comment by killyouslow** [comment]
naw you got it all wrong oloflarsson, i pay you to use your premium plugins, THOUSANDS of people base there servers on your plugins that are suppose to WORK PROPERLY,

The way I SEE IT is that you work for US, not the other way around, and i ASSURE YOU that all my frustrations are VALID!

You cant break your plugins after every single update in one way or another and expect people to be happy about it, i've paid for your premium resources on spigot, I've implemented and supported your plugins for years, and quite frankly i'm feed up with being "Respectful" aswell as patient!

**Comment by ulumulu1510** [comment]
@killyouslow

I am not sure why you spread your foul words and missinformation over so many github issues but one thing is certain:

With your attitude, you don't get anywhere.


Fact is, that Cayorion developes plugins foremost for his own usage and he does grant others to use some of this work. This work is subject to changes, as everything is, and one of these changes was to disable the usys since he had no use for it anymore. (The world exception set proved to be more usefull)

As a person I can understand that you are unhappy and miss certain features. As a developer I cannot understand the attitude you aproach us with. You don't seem to understand that developers don't owe you anything. Not changes, nor new features.

Last, but not least, the "failures" you pointed out only evolve from your lack of perspective and gratefullness.

Tl;DR:
Watch your attitude and stop posting spite all over the issue tracker.

**Comment by killyouslow** [comment]
I've already commented on this post in a previous post, but i will point out that just because you think i have a "lack of perspective and gratefulness" were the project managers / developers are concerned doesn't mean that you don't share in a lack of perspective and gratefulness were the community that supported you is concerned!
